src/black/__init__.py

```diff
@@ -511,6 +520,29 @@ def path_empty(
         ctx.exit(0)
 
 
+def reformat_code(
+    content: str, fast: bool, write_back: WriteBack, mode: Mode, report: Report
+) -> None:
+    """
+    Reformat and print out `content` without spawning child processes.
+    Similar to `reformat_one`, but for string content.
+
+    `fast`, `write_back`, and `mode` options are passed to
+    :func:`format_file_in_place` or :func:`format_stdin_to_stdout`.
+    """
+    try:
+        changed = Changed.NO
+        if format_stdin_to_stdout(
+            content=content, fast=fast, write_back=write_back, mode=mode
+        ):
+            changed = Changed.YES
+        report.done(content, changed)
```

Comment:
This will make it print something like "<all of your code> already well formatted". That's going to be pretty ugly if you input multiple lines. I'd suggest passing something like `Path("<string>")`, matching what Python itself does:
```
% python -c 'def f(x): 1/0
quote> f(1)
quote> '''
Traceback (most recent call last):
  File "<string>", line 2, in <module>
  File "<string>", line 1, in f
ZeroDivisionError: division by zero
```

Reply:
Just applied this change myself so @cooperlees can do a release if he wants

Reply:
Resolved in 39c3ae1.